### PR TITLE
Dynamic commands

### PR DESCRIPTION
**Pull request by isaacy13**
- changes DB to support dynamic commands (e.g.: running tests on all visible objects)
- GUI update for table (organizes results based on object test was run on)
- GUI update for table (now only bottom border shows)
- Command builder (getCMD()) was changed to support dynamic commands
- Status bar message updated to say which object is currently being tested

### CODE REVIEW

**Comment by isaacy13** [comment]
fixed bugs stemming from merge conflict

File: include/VerificationValidation.h [hunk 1]
```diff
@@ -8,40 +8,106 @@
...
+
+            std::vector<Arg> lhsArgList(ArgList);
+            std::vector<Arg> rhsArgList(rhs.ArgList);
+            std::sort(lhsArgList.begin(), lhsArgList.end());
+            std::sort(rhsArgList.begin(), rhsArgList.end());
+
+            for (int i = 0; i < lhsArgList.size(); i++) {
+                if (lhsArgList[i].type == rhsArgList[i].type && (lhsArgList[i].type == Arg::Type::ObjectName || lhsArgList[i].type == Arg::Type::ObjectNone || lhsArgList[i].type == Arg::Type::ObjectPath))
+                    continue;
+                if (lhsArgList[i].argument != rhsArgList[i].argument)
```
Comment (daperego) [inline include/VerificationValidation.h]: Need to also compare defaultValue -- noticed an error where No Overlap tests were being run as No Null Region tests because the arguments are identical but the default values are not.

**Review by daperego — CHANGES_REQUESTED** [review]
Fix comment

File: include/VerificationValidation.h (hunk 1 above)
Comment (isaacy13) [reply include/VerificationValidation.h]: great catch, changes pushed

**Review by daperego — APPROVED** [review]
